core/templates/pages/topic-editor-page/editor-tab/topic-editor-stories-list.directive.html

```diff
@@ -30,7 +30,9 @@
       </a>
     </td>
     <td>
-      <span ng-click="deleteCanonicalStory(summary.getId())" class="fas fa-trash-alt list-summary"> </span>
+      <a ng-click="deleteCanonicalStory(summary.getId())" class="list-summary">
+        <span class="fas fa-trash-alt"> </span>
```

Comment:
Remove the space in between tag (Like the way you have done in the other places.)

Reply:
Done.